src/strings.c

```diff
@@ -297,12 +298,23 @@ STRING_HANDLE STRING_new_JSON(const char* source)
         }
         else
         {
-            if ((result = (STRING*)malloc(sizeof(STRING))) == NULL)
+            //size_t malloc_len = vlen + 5 * nControlCharacters + nEscapeCharacters + 3;
```

Comment:
Should remove?

Reply:
No, I want it there to show the intended math using safe macros